libs/clients/syslumenn/src/lib/syslumennClient.config.ts

```diff
@@ -17,7 +17,7 @@ export const SyslumennClientConfig = defineConfig({
     return {
       url: env.required('SYSLUMENN_HOST', 'https://api.syslumenn.is/dev'),
       fetch: {
-        timeout: env.optionalJSON('SYSLUMENN_TIMEOUT') ?? 10000,
+        timeout: env.optionalJSON('SYSLUMENN_TIMEOUT') ?? 100000,
```

Comment:
Does the linter allow something like `10_000` which is a nice way to process numbers this big mentally?